packages/filebrowser/src/model.ts

```diff
@@ -619,18 +615,6 @@ export class FileBrowserModel implements IDisposable {
     });
   }
 
-  /**
-   * Handle internal model refresh logic.
-   */
-  private _scheduleUpdate(): void {
-    let date = new Date().getTime();
-    if (date - this._lastRefresh > MIN_REFRESH) {
-      void this.refresh();
-    } else {
-      this._requested = true;
```

Comment:
ah, so it actually was doing a debounce, in that if we tried to schedule an update too soon, it would note it and follow up with an update at some point in the future.

Reply:
Yes, it was doing that.

Note, however, that the user clicking `refresh()` circumvents the polling all-together, so they get immediate feedback.

Reply:
Right, the original intent was for the auto-poll to not happen too soon after a user or programmatically triggered poll.

Reply:
Then we should be okay with deleting the logic entirely, then. If a user clicking on the refresh button calls the poll refresh, the poll knows about the refresh and its interval starts over again.